internal/api/grpc/user/v2/user.go

```diff
@@ -485,9 +408,98 @@ func (s *Server) HumanMFAInitSkipped(ctx context.Context, req *user.HumanMFAInit
 }
 
 func (s *Server) CreateUser(ctx context.Context, req *user.CreateUserRequest) (*user.CreateUserResponse, error) {
-	return nil, zerrors.ThrowUnimplemented(nil, "", "not implemented")
+	orgId := req.GetOrganizationId()
+	if err := s.command.CheckOrgExists(ctx, orgId); err != nil {
+		return nil, err
+	}
+	switch userType := req.GetUserType().(type) {
+	case *user.CreateUserRequest_Human_:
```

Comment:
please put the human and machine case into a separate function
IMO it's hard to read the function (i can't even display the whole function on a single screen)